common/cache/src/query/local_cache.rs

```diff
@@ -35,6 +36,10 @@ const CACHE_READ_BYTES_FROM_LOCAL: &str = "cache_read_bytes_from_local";
 const CACHE_ACCESS_COUNT: &str = "cache_access_count";
 const CACHE_ACCESS_HIT_COUNT: &str = "cache_access_hit_count";
 
+const FUSE_TBL_BLOCK_PREFIX: &str = "_b";
```

Comment:
Where these prefix original defines? It would be better to use the original definitions instead defining them.

Reply:
it seem not that elegant because original definitions is in `query/storages/fuse/constants`...
so I repeat it here  : (

Reply:
I think the location is string is simple, not sure may it can be a structure contain s a type here.
Wondering how the location get from?

Reply:
The prefix type is a underlying logic, the up-front can't defines it, so we need a place tell us what's the location type exactly is.

Reply:
> 

we will get block path from `segments`, defines here: 
https://github.com/datafuselabs/databend/blob/170084277c/query/src/storages/fuse/meta/segment.rs#L21-L26

Reply:
Does the QueryCache mainly used by the fuse engine?
 If so we can move it to the storages/fuse folder.